src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Location/DisplayDeviceLocation/readme.metadata.json

```diff
@@ -20,8 +20,7 @@
     },
     "offline_data": [],
     "redirect_from": [
-        "/net/latest/wpf/sample-code/displaydevicelocation.htm",
-        "/net/wpf/sample-code/display-device-location/"
+        "/net/latest/wpf/sample-code/displaydevicelocation.htm"
     ],
```

Comment:
Was this an intentional change?

Reply:
Not intentional. I had the display device location issue resolved in my other PR, forgot about it here.